src/shared/utils.js

```diff
@@ -18,4 +18,8 @@ export function callAfter(fn, i) {
 	return () => {
 		if (!--i) fn();
 	};
+}
+
+export function addLoc(element, file, line, column, char) {
+	element.__svelte_meta = { file, line, column, char };
```

Comment:
I guess I was imagining that the loc data would live down inside an object under some key under `__svelte_meta`, to keep it conceptually separated from other future non-loc data.

Reply:
ah yeah, that probably makes sense